lib/reconcile/rabbitmq.go

```diff
@@ -323,7 +300,7 @@ func getRabbitMQEnvVars(statefulSetName string, cr *apiv1alpha1.Astarte) []v1.En
 
 func getRabbitMQPodSpec(statefulSetName, dataVolumeName string, cr *apiv1alpha1.Astarte) v1.PodSpec {
 	serviceAccountName := statefulSetName
-	if pointy.BoolValue(cr.Spec.RBAC, false) {
+	if !cr.Spec.RBAC {
```

Comment:
Yes, this code was broken and the only reason this used to work was because RBAC was never set (and its default was in fact true, not false). Now the boolean logic is correct